src/problems/TallBoxSf/Makefile

```
@@ -0,0 +1,12 @@
+SHELL := /bin/zsh
+
+# Sigma13-Z1, solar neighborhood
+# Sigma_gas = 13.0 M_sun/pc^2
+# Sigma_star = 42.0 M_sun/pc^2
+# sigma_1 = 7.0 km/s
+# rho_dm = 6.4e-3 M_sun/pc^3
+# R0 = 8000.0 pc
+# z_star = 245.0 pc (default)
+run1:
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)